client/layout/masterbar/style.scss

```diff
@@ -564,6 +640,14 @@ $autobar-height: 20px;
 		transform: scale3d( 1.15, 1.15, 1.15 );
 	}
 }
+@keyframes mobile-menu-fade-in {
```

Comment:
We do have an existing `appear` animation already that does essentially the same:

https://github.com/Automattic/wp-calypso/blob/9f3432741d318601ae081858ab2de6f55a5263eb/client/assets/stylesheets/shared/_animation.scss#L6